Material.Blazor/Components/DatePicker/InternalDatePickerDayButton.razor

```diff
@@ -13,4 +13,5 @@
           class="@classes"
           Disabled="@ButtonDisabled"
           Label="@Day"
-          @onclick="@OnClickAsync" />
\ No newline at end of file
+          @onclick="@OnClickAsync"
+          type="button" />
```

Comment:
Out of curiosity: what's the issue when we don't include `type="button"`?

Reply:
Now I'm trying to remember. I *think* it was about having a DatePicker in a form. If you don'tuse `type="button"` as recall the form interprets a button click as a submit action.